playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: Backup etcd
   include: ./etcd/backup.yml
-  when: openshift_upgrade_skip_etcd_backup | default(false) | bool
+  when: not openshift_upgrade_skip_etcd_backup | default(false) | bool
```

Comment:
Should this be named `openshift_upgrade_backup_etcd` instead, with a default value of `true`. We'd have to support a renaming at least temporarily, but at least the future logic won't continue to make my head hurt.